databases/week1/homework_week1_Database.sql

```diff
@@ -0,0 +1,54 @@
+# Database week1 HomeWork by Adnan Aziz Class_17
+
+# 1-Find out how many tasks are in the task table
+select count(task.id)
+from task;
```

Comment:
You could  actually do like this `SELECT COUNT(id) FROM task;`   to make your code short and easy to understand.  You don't need to do like `task.id` or` task.title `when you are only dealing with a single table. So the same applies to quetions nr.1 to nr.7.

Reply:
i have the changes and now going to close the pull request